go/service/contacts_test.go

```diff
@@ -76,10 +79,19 @@ func TestContactSyncAndSearch(t *testing.T) {
 	// will not. We are expecting to be able to search for both, but not see
 	// both in user recommendations.
 
-	err := all.contactsHandler.SaveContactList(context.Background(), keybase1.SaveContactListArg{
+	_, err := all.contactsHandler.SaveContactList(context.Background(), keybase1.SaveContactListArg{
+		Contacts: rawContacts,
+	})
+	require.NoError(t, err)
+
+	// bust cache, new resolution should be returned
+	clock.Advance(72 * time.Hour)
+	all.contactsMock.PhoneNumbers["+48111222333"] = contacts.MakeMockLookupUser("alice", "")
+	newlyResolved, err := all.contactsHandler.SaveContactList(context.Background(), keybase1.SaveContactListArg{
 		Contacts: rawContacts,
 	})
 	require.NoError(t, err)
+	require.Len(t, newlyResolved, 1)
```

Comment:
can you do require.Equal to see if the newly resolved is the right one? checking contact name, username, and assertion should be enough.